src/js/angular/core/services/jwt-auth.service.js

```diff
@@ -228,12 +221,9 @@ angular.module('graphdb.framework.core.services.jwtauth', [
                 return this.hasOverrideAuth && this.principal && this.principal.username === 'overrideauth';
             };
 
-            this.getAuthToken = function () {
-                return this.auth;
-            };
-
             this.authTokenIsType = function (type) {
-                return this.auth && this.auth.startsWith(type);
+                const token = AuthTokenService.getAuthToken();
+                return token && token.startsWith(type);
```

Comment:
It'd have been better if we moved all such checks inside the service: `token.startsWith('GDB')`, `token.startsWith('Bearer')`, etc.